examples/DotNetCore/WebJobs/WebJobHelloWorld/WebJobHelloWorld/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Configuration;
+
+namespace WebJobHelloWorld
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var hostBuilder = new HostBuilder();
+            hostBuilder.ConfigureHostConfiguration((config) =>
+            {
+                LoadConfiguration(config);
+            });
+            hostBuilder.ConfigureWebJobs(b =>
+            {
+                b.AddAzureStorageCoreServices();
+                b.AddAzureStorage();
+            });
+            hostBuilder.ConfigureLogging((context, b) =>
+            {
+                b.AddConsole();
+            });
+
+            var host = hostBuilder.Build();
+            using (host)
+            {
+                host.Run();
```

Comment:
Just for my education, what is this webjob doing?

Reply:
AzureWebJobs need a Storage Account and a Queue Name to queue events. This web job tries to read it from local configuration (appSettings.json or environment variables), or read it from the config store.